### DISCUSSION TITLE
Does this violate the terms of use of Netflix?

### DISCUSSION THREAD

**Discussion by Aarontheissueguy** [CastagnaIT/plugin.video.netflix#1105] [General]
Hello everyone, I've recently started using this add on and am very happy with it. I am still somewhat concerned though if I risk my account being blocked because it's a family account used by multiple people. Is there any risk of that happening?

**Comment by jakermx** [comment]
The account limits are set by the concurrent ESN sessions, based on this, you can have X devices using the same account, while you keep  respect your paid plan's device number as concurrent PLAYING  devices.

Before any blocking action , you will receive , 10 email, when a nre dedvice goes over your plan limit...

Keep in mind that , this addOn is a WebBased implementation of the NF Client...so think you are on your PC Browser

Cheers
